### PR TITLE
feat product list for seller done

### PR DESCRIPTION
**Pull request by gluhovSemen**

### CODE REVIEW

File: src/apps/inventory/models.py [hunk 1]
```diff
@@ -14,6 +16,12 @@ class Product(models.Model):
     description = models.TextField(blank=True)
     category = models.ForeignKey(Category, on_delete=models.PROTECT)
     price = models.DecimalField(max_digits=10, decimal_places=2)
+    seller = models.ForeignKey(
```
Comment (egorvavilov) [inline src/apps/inventory/models.py:18]: Maybe lets call it "created_by". Please also add information about when product was created.

File: src/apps/inventory/models.py [hunk 2]
```diff
@@ -14,6 +16,12 @@ class Product(models.Model):
...
+        User,
+        on_delete=models.CASCADE,
+        related_name="products",
+        limit_choices_to={"user_type": "Seller"},
```
Comment (egorvavilov) [inline src/apps/inventory/models.py:22]: love it

File: src/apps/inventory/views.py [hunk 3]
```diff
@@ -1,7 +1,18 @@
...
+
+
+class ProductBySellerListView(ListView):
+    model = Product
+    paginate_by = 10
+
+    def get_queryset(self):
+        if self.request.user.is_authenticated:
+            return Product.objects.filter(seller=self.request.user)
+        else:
```
Comment (egorvavilov) [inline src/apps/inventory/views.py]: Do we really need to have else here?)

**Review by egorvavilov — CHANGES_REQUESTED** [review]

File: src/apps/inventory/views.py (hunk 3 above)
Comment (gluhovSemen) [reply src/apps/inventory/views.py]: Fair

File: src/apps/inventory/models.py (hunk 1 above)
Comment (gluhovSemen) [reply src/apps/inventory/models.py:18]: to be honest i like seller more, created_by is a little misleading maybe "in the future" we will add this field for example for the country where the product was created or the brand of manufacturer

Comment (gluhovSemen) [reply src/apps/inventory/models.py:18]: Time added

File: src/apps/inventory/models.py [hunk 4]
```diff
@@ -1,5 +1,7 @@
 from django.db import models
 
+from src.apps.user.models import User
```
Comment (egorvavilov) [inline src/apps/inventory/models.py]: Import User via get_user_model()

when you create Foreign key on user model  it is better to use settings.AUTH_USER_MODEL

File: src/apps/inventory/views.py [hunk 5]
```diff
@@ -1,7 +1,16 @@
...
     model = Product
+
+
+class ProductBySellerListView(ListView):
+    model = Product
+    paginate_by = 10
+
+    def get_queryset(self):
+        if self.request.user.is_authenticated:
+            return Product.objects.filter(seller=self.request.user)
```
Comment (egorvavilov) [inline src/apps/inventory/views.py:16]: return ...none() we still need it

**Review by egorvavilov — CHANGES_REQUESTED** [review]

File: src/apps/inventory/views.py (hunk 5 above)
Comment (gluhovSemen) [reply src/apps/inventory/views.py:16]: I dont think so, i have tested it, it works fine without this else statement

**Review by egorvavilov — APPROVED** [review]

**Pull request merged**